packages/core/src/components/StepList/Step.jsx

```diff
@@ -0,0 +1,82 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import StepLink from './StepLink';
+import SubStep from './SubStep';
+import classNames from 'classnames';
+import { stepShape } from './StepList';
+
+export const Step = ({ step, ...props }) => {
+  const start = step.isNextStep;
+  const resume = step.started && !step.completed;
+  const className = classNames('ds-c-step', {
+    'ds-c-step--current': start || resume,
+    'ds-c-step--completed': step.completed
+  });
+  return (
+    <li className={className}>
+      <h2 className="ds-c-step__header ds-h3 ds-u-display--flex ds-u-justify-content--between">
+        <div className="ds-c-step__title">{step.title}</div>
+        <div>
+          {step.completed && (
+            <div className="ds-c-step__completed">
+              <span>{props.completedText}</span>
+              {!step.steps && (
+                <StepLink
+                  route={step.route}
+                  stepId={step.id}
+                  screenReaderText={`"${step.title}"`}
```

Comment:
This is because of the discussion above with Trevor. I didn't want to have to send in five strings representing the different sentences, so I thought it would be easier to have the screen reader read 'Resume "Choose a tax year"'

Reply:
I concur with @pwolfert about not wanting to pass in multiple strings. When he explained the logistics of it, I realized I was asking a lot for contextual differences. A string like "Resume Choose a tax year" or "Edit Choose a tax year", is pretty clear, and more actionable than just Resume, even if it's not a completely custom block of micro-content.